pilot.go

```diff
@@ -134,6 +137,74 @@ func (c *chanController) SpliceOut(chanPoint *wire.OutPoint,
 	return nil, nil
 }
 
+// FeeEstimate returns the total estimated fee needed for the autopilot to open
+// channels of the given sizes. Since we currently open each channel in a
+// separate tx, the combined fee estimate will just be the sum of the
+// individual ones.
+func (c *chanController) FeeEstimate(chanSizes []btcutil.Amount) (
+	btcutil.Amount, error) {
+
+	// Find all unlocked unspent witness outputs that satisfy the minimum
+	// number of confirmations required.
+	coins, err := c.server.cc.wallet.ListUnspentWitness(
+		c.minConfs, math.MaxInt32,
+	)
+	if err != nil {
+		return 0, err
+	}
+
+	// Since we currently open each channel in a separate tx, the combined
+	// fee estimate will just be the sum of the individual ones. We attempt
+	// to mimic the coin selection done in the wallet. This won't be
+	// totally accurate (coins can be selected differently when the
+	// channels are actually being opened), but this is just an estimate
+	// soooo ¯\_(ツ)_/¯
+	var totalWeight int64
+	currentCoin := 0
+	for _, amt := range chanSizes {
```

Comment:
This assumes a _single_ transaction for all channels that has a funding and change address. As a result, it'll underestimate since it's missing the signalling overhead for teach of the individual transactions.

Reply:
What overhead are you referring to? This is how we do estimates on transaction weights in other parts of the codebase.

Reply:
Overhead as in things like the transaction version and lock time. In other parts, when we do estimation it's for a single transaction. This attempts to estimate the weight of a single mega funding transaction, but ultimately a series of funding transactions will be created instead. 

In terms of fixing the issue at hand (more breathing room for fees), this won't materially affect that, but it will cause the estimates to be off.